litmus-portal/frontend/src/views/AnalyticsDashboard/MonitoringDashboardPage/ChaosTable/index.tsx

```diff
@@ -0,0 +1,167 @@
+                  {chaosList.length ? (
+                    chaosList
+                      .slice(0)
+                      .slice(
+                        page * rowsPerPage,
+                        page * rowsPerPage + rowsPerPage
+                      )
+                      .map((data: ChaosEventDetails, index: number) => {
+                        const isItemSelected = isSelected(data.id);
+                        const labelId = `enhanced-table-checkbox-${index}`;
+                        return (
+                          <TableRow
+                            hover
+                            onClick={() => {
+                              handleClick(data.id);
+                            }}
+                            role="checkbox"
+                            aria-checked={isItemSelected}
+                            tabIndex={-1}
+                            key={data.id}
+                            selected={isItemSelected}
+                          >
+                            <TableData
+                              data={data}
+                              itemSelectionStatus={isItemSelected}
+                              labelIdentifier={labelId}
+                            />
+                          </TableRow>
+                        );
+                      })
+                  ) : (
+                    <TableRow>
+                      <TableCell colSpan={6}>
+                        <div className={classes.noRecords}>
+                          <img
+                            src="/icons/cloudIcon.svg"
+                            className={classes.cloudIcon}
+                            alt="Chaos cloud"
+                          />
+                          <Typography
+                            align="center"
+                            className={classes.noRecordsText}
+                          >
+                            {t(
+                              'analyticsDashboard.monitoringDashboardPage.chaosTable.noRecords'
+                            )}
+                          </Typography>
+                        </div>
+                      </TableCell>
+                    </TableRow>
+                  )}
+                  {chaosList.length > 0 && emptyRows > 0 && (
+                    <TableRow style={{ height: 75 * emptyRows }}>
+                      <TableCell colSpan={6} />
+                    </TableRow>
+                  )}
+                </TableBody>
+              </Table>
+            </TableContainer>
+            {chaosList.length ? (
+              <TablePagination
+                rowsPerPageOptions={[5, 10, 25, 50]}
+                component="div"
+                count={chaosList.length}
+                rowsPerPage={rowsPerPage}
+                page={page}
+                onChangePage={handleChangePage}
+                onChangeRowsPerPage={handleChangeRowsPerPage}
+              />
+            ) : (
+              <div />
```

Comment:
is this empty div required...can't the logic be changed to
```
chaosList.length && (
...
)
```

Reply:
> 0 also i am adding as numeric value will render by the suggested change. updated this as suggested with minor addition.

Reply:
Done